search-app/src/cmr/search/api/tags.clj

```diff
@@ -5,7 +5,8 @@
    [clojure.string :as string]
    [cmr.acl.core :as acl]
    [cmr.common-app.api.enabled :as common-enabled]
-   [cmr.common.log :refer (debug info warn error)]
+   [cmr.common.log :refer (debug)]
```

Comment:
do you still need () around debug?

Reply:
you do. It won't work without the parenthesis